### DISCUSSION TITLE
Citation for bed_reader

### DISCUSSION THREAD

**Discussion by jeromekelleher** [fastlmm/bed-reader#16] [Q&A] [answered]
Is there a preferred citation for referring to bed_reader in published work?

**Comment by CarlKCarlK** [comment, marked as answer]
Thanks for asking! This would be fine:
Carl Kadie and Christian Widmer. "Bed-Reader." GitHub, https://github.com/fastlmm/bed-reader/


  *   Carl


From: Jerome Kelleher ***@***.***>
Sent: Friday, August 4, 2023 8:44 AM
To: fastlmm/bed-reader ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [fastlmm/bed-reader] Citation for bed_reader (Discussion #16)
Importance: High


Is there a preferred citation for referring to bed_reader in published work?

-
